dspace-api/src/main/resources/org/dspace/storage/rdbms/sqlmigration/postgres/V5.7_2017.04.11__DS-3563_Index_metadatavalue_resource_type_id_column.sql

```diff
@@ -12,5 +12,5 @@
 -- Create an index on the metadata value resource_type_id column so that it can be searched efficiently.
 
 DROP INDEX IF EXISTS metadatavalue_resource_type_id_idx;
-
-CREATE INDEX metadatavalue_resource_type_id_idx ON metadatavalue (resource_type_id);
+-- disable follow line because it broke upgrade to the Configurable Workflow in DSpace 7
```

Comment:
We should not edit any of the existing Flyway scripts. Can you provide more information on the "broke upgrade to the Configurable Workflow in DSpace 7" error you encountered?